### PR TITLE
Fix some bugs for applying OEMQA board

### PR DESCRIPTION
**Pull request by baconYao**
Fix bugs

- #50 
- #51 
- #52

### CODE REVIEW

File: API/Jira/scenarios/pc/pc.py [hunk 1]
```diff
@@ -470,23 +471,23 @@ def _create_rts_task(self, story_task={}):
                     self.project_profile['labels']['rts_labels'][idx]
                 )
 
+            # Assign Epic
+            fields['parent'] = {
+                'id': self.jira_api.jira_project['epic'][self.epic]
+            }
+
             # FIXME: Use regex to verify the valid time format
```
Comment (stanley31huang) [inline API/Jira/scenarios/pc/pc.py:479]: could we remove all of marked code here?

**Review by stanley31huang — CHANGES_REQUESTED** [review]
please see my inline comments, thanks

File: API/Jira/scenarios/pc/pc.py (hunk 1 above)
Comment (baconYao) [reply API/Jira/scenarios/pc/pc.py]: Remove those comment block. Please check it again.

**Review by weizhenwu — APPROVED** [review]
LGTM

**Review by stanley31huang — APPROVED** [review]
LGTM

**Pull request merged**